cluster/gce/container-linux/configure-helper.sh

```diff
@@ -1293,22 +1293,32 @@ EOF
     setup-addon-manifests "addons" "metrics-server"
   fi
   if [[ "${ENABLE_CLUSTER_DNS:-}" == "true" ]]; then
-    setup-addon-manifests "addons" "dns"
-    local -r kubedns_file="${dst_dir}/dns/kube-dns.yaml"
-    mv "${dst_dir}/dns/kube-dns.yaml.in" "${kubedns_file}"
-    if [ -n "${CUSTOM_KUBE_DNS_YAML:-}" ]; then
-      # Replace with custom GKE kube-dns deployment.
-      cat > "${kubedns_file}" <<EOF
+    if [[ "${CLUSTER_DNS_CORE_DNS:-}" == "true" ]]; then
```

Comment:
can you split this off into two bash functions

setup-coredns-manifest()
setup-kube-dns-manifest()

to make it easier to read